metricbeat-tests/configurations/apache.yml

```diff
@@ -16,6 +16,12 @@ metricbeat.modules:
   # Password of hosts. Empty by default
   #password: password
 
-output.file:
-  path: "/tmp"
-  filename: "${FILE_NAME}.metrics"
\ No newline at end of file
+output.elasticsearch:
+  hosts: ["http://elasticsearch:9200"]
+  username: "elastic"
+  password: "p4ssw0rd"
+
+setup.kibana:
+  host: "http://kibana:5601"
+  username: "elastic"
+  password: "p4ssw0rd"
```

Comment:
I think it'd be better to have this configuration as part of the metricbeat execution and not as part of the test files. These settings can be also set as execution flags, as `-E output.elasticsearch.hosts=http://elasticsearch:9200 -E output.elasticsearch.username=elastic ...`
This way we could also consider different scenarios with the same configuration files.

Kibana options shouldn't be needed for ingestion. We might avoid any test with kibana by now, I think it would only make sense if we test dashboards.

Reply:
Yeah, first idea was to configure the entire metricbeat with env vars, which would be more flexible. If possible, I'd go with that direcction